### PR TITLE
Add polish version of the questions

### PR DESCRIPTION
**Pull request by Shoulao**
Hey @lydiahallie!

Here you have the questions 1-124 in polish!

### CODE REVIEW

**Comment by lydiahallie** [comment]
Thanks so much! Could you maybe provide a link to the polish translation in the original English readmes?

**Comment by Shoulao** [comment]
> Thanks so much! Could you maybe provide a link to the polish translation in the original English readmes?

Done

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ List of available languages:
 * [Français](./fr-FR/README_fr-FR.md)
 * [日本語](./ja-JA/README-ja_JA.md)  
 * [한국어](./ko-KR/README-ko_KR.md) 
+* [Polish](../pl-PL/README-pl_PL.md)
```
Comment (lydiahallie) [inline README.md]:
```suggestion
* [Polish](./pl-PL/README-pl_PL.md)
```

**Comment by Shoulao** [comment]
My bad, I was doing this in a hurry

File: pl-PL/README-pl_PL.md [hunk 2]
```diff
@@ -0,0 +1,4234 @@
...
+#### Odpowiedź: A
+
+Funkcja `add` zwraca funkcję strzałkową, która zwraca funkcję strzałkową, która zwraca funkcję strzałkową. Pierwsza funkcja otrzymuje argument `x` z wartością `4`. Wywołujemy drugą funkcję, która otrzymuje argument `y` z wartością `5`. Następnie wywołujemy trzecią funkcję, która otrzymuje argument `z` z wartością `6`. Kiedy staramy się dostać do wartości `x`, `y` i `z` wewnątrz ostatniej funkcji strzałkowej, silnika JS idzie w górę łańcucha zasięgu, by odszukać wartości dla `x` i `y`. W rezultacie otrzymujemy `4` `5` `6`.
+
+</p>
+</details>
+
+---
+
+###### TBC ----------->
```
Comment (jakeherp) [inline pl-PL/README-pl_PL.md:3991]: If all following questions were not translated yet, can you remove them for now to avoid confusion?

**Review by jakeherp — CHANGES_REQUESTED** [review]
See below:

**Comment by jakeherp** [comment]
Closing due to inactivity.

**Pull request closed without merging**